build/release/Makefile

```diff
@@ -160,16 +160,14 @@ build.image.$(1).$(2).$(3):
 build.all.images: build.image.$(1).$(2).$(3)
 publish.image.$(1).$(2).$(3): ; @docker push $(1)/$(2)-$(3):$(VERSION)
 publish.all.images: publish.image.$(1).$(2).$(3)
+# tag the master image, but do not tag the release image with a generic channel tag
 promote.image.$(1).$(2).$(3):
 	@docker pull $(1)/$(2)-$(3):$(VERSION)
-	@[ "$(CHANNEL)" = "master" ] || docker tag $(1)/$(2)-$(3):$(VERSION) $(1)/$(2)-$(3):$(VERSION)-$(CHANNEL)
 	@docker tag $(1)/$(2)-$(3):$(VERSION) $(1)/$(2)-$(3):$(CHANNEL)
-	@[ "$(CHANNEL)" = "master" ] || docker push $(1)/$(2)-$(3):$(VERSION)-$(CHANNEL)
-	@docker push $(1)/$(2)-$(3):$(CHANNEL)
+	@[ "$(CHANNEL)" = "release" ] || docker push $(1)/$(2)-$(3):$(CHANNEL)
```

Comment:
I don't follow why these changes are needed.

Reply:
The background for these changes is in the details of [these notes](https://github.com/rook/rook/issues/1885#issuecomment-460444455). To summarize:
- Since there is only a single release channel, there is no need for the channel suffix on the image name. For example, why do we need `v1.0.0-release`?
- Regarding removal of the `release` tag (for example `rook/ceph:release`): To carefully control the versioning of upgrades, it is not recommended to use a tag that changes be used. This is one of the reasons why the `master` tag is regularly a problem for users... because they don't realize when the image changes and it breaks things.